### PR TITLE
Remove Gecko-specific Python bootstrap logic

### PR DESCRIPTION
**Pull request by mrobinson**
Servo is no longer completely vendored into Gecko. Instead parts of
Gecko are vendored into Servo. This change removes Python mach bootstrap
code that was written to accommodate the previous situation. It's no
longer necessary.

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes do not require tests because this just removes unused code from build scripts.

### CODE REVIEW

File: python/mach_bootstrap.py [hunk 1]
```diff
@@ -184,8 +167,8 @@ def _activate_virtualenv(topdir, is_firefox):
     # and it will check for conflicts.
     requirements_paths = [
         os.path.join("python", "requirements.txt"),
-        wptrunner_path(is_firefox, topdir, "requirements.txt",),
-        wptrunner_path(is_firefox, topdir, "requirements_firefox.txt"),
+        os.path.join(WPT_RUNNER_PATH, "requirements.txt",),
+        os.path.join(WPT_RUNNER_PATH, "requirements_firefox.txt"),
```
Comment (mukilan) [inline python/mach_bootstrap.py:171]: Is `requirements_firefox.txt` still needed?

**Review by mukilan — APPROVED** [review]

File: python/mach_bootstrap.py (hunk 1 above)
Comment (mrobinson) [reply python/mach_bootstrap.py:171]: Great question. I've kicked off a try job, attempting to remove that. It's hard to say which requirements inside of the Servo WPT executor might be accidentally relying on Gecko's list of packages.

**Comment by mrobinson** [comment]
Looks like we can safely remove this. Sending this one to the merge queue. Thanks!

**Pull request merged**